api/inventorypb/agents.proto

```diff
@@ -623,6 +623,12 @@ message AddMongoDBExporterRequest {
   bool tls = 5;
   // Skip TLS certificate and hostname validation.
   bool tls_skip_verify = 6;
+  // Certificate file.
+  string tls_certificate_key_file = 10;
```

Comment:
Not file – file's content. So API field (unlike pmm-admin CLI flag, for example) should not have "_file".

Reply:
Yes, you are right. Will remove "_file" from fields, where I have already loaded certificate.